### DISCUSSION TITLE
Pagination failing for list likes

### DISCUSSION THREAD

**Discussion by jazeller** [trakt/trakt-api#295] [Q&A]
Hi team, following the [documentation here](https://trakt.docs.apiary.io/#reference/lists/list-likes/get-all-users-who-liked-a-list) to get all likes from a list, I'm making this call:

'https://api.trakt.tv/lists/zzz/likes?limit=9999' where zzz is replaced by a numerical list id.

I'm only getting a max of 25 likes back regardless of the actual number of likes. When I look at the headers that are returned, there are no X-Pagination-Page or X-Pagination-Limit fields. See below for returned headers for a call using list ID  11480710 which has 36 likes. The returned json has 25 items. I have also tried removing the "?limit..." at the end of the query, and also tried adding "?page=2" instead. All three attempts return the same 25 likes. I have also tried this with a different list id (e.g. 9849557, which has 29 likes) and get the same behavior. 

Is there a way to get the rest of the likes?

{'Date': 'Wed, 23 Jun 2021 21:47:30 GMT',
 'Content-Type': 'application/json; charset=utf-8',
 'Transfer-Encoding': 'chunked',
 'Connection': 'close',
 'X-Frame-Options': 'SAMEORIGIN',
 'X-XSS-Protection': '1; mode=block',
 'X-Content-Type-Options': 'nosniff',
 'X-Download-Options': 'noopen',
 'X-Permitted-Cross-Domain-Policies': 'none',
 'Referrer-Policy': 'strict-origin-when-cross-origin',
 'Expires': 'Wed, 23 Jun 2021 22:02:34 GMT',
 'X-Pagination-Page-Count': '2',
 'X-Pagination-Item-Count': '36',
 'Cache-Control': 'public, max-age=3600',
 'Vary': 'Accept-Encoding',
 'X-Ratelimit': '{"name":"UNAUTHED_API_GET_LIMIT","period":300,"limit":1000,"remaining":999,"until":"2021-06-23T21:05:00Z"}',
 'ETag': 'zzz"',
 'X-Request-Id': 'zzz',
 'X-Runtime': '0.025531',
 'CF-Cache-Status': 'HIT',
 'Age': '2696',
 'cf-request-id': 'zzz',
 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"',
 'Server': 'cloudflare',
 'CF-RAY': '6640ea4b0a552544-SJC'}

**Comment by mfederowicz** [comment]
interesting one day to add that feature, but now tested pagination @rectifyer what about it? ok or maybe something is broken in after year (or is it depends on some other flags which standard account dont have)?
